libff/algebra/curves/tests/test_bilinearity.cpp

```diff
@@ -126,6 +131,7 @@ template<typename ppT> void affine_pairing_test()
     ASSERT_EQ((ans1 ^ Fr<ppT>::field_char()), GT_one);
 }
 
+#if 1 // BLS12_381 debug (VV)
```

Comment:
similarly, we can probably remove this now.

Reply:
Done.